docs/version0.19.md

```diff
@@ -44,6 +45,10 @@ The latest builds of OpenJDK with OpenJ9 for Java 8 and 11 at the AdoptOpenJDK c
 
 To learn more about support for OpenJ9 releases, including OpenJDK levels and platform support, see [Supported environments](openj9_support.md).
 
+### stringbuffer-and-stringbuilder-above-1-g-grow-to-the-maximum-size
```

Comment:
- I think you copied the link string here, rather than the title!
- Please add `![Start of content that applies to Java 8](cr/java8.png)` at beginning of both titles (doesn't affect link string). See last item in Version 16 release notes for an example.

Reply:
Fixed.